ui/apps/platform/src/Containers/Compliance/widgets/StandardsAcrossEntity.js

```diff
@@ -22,10 +22,13 @@ function formatAsPercent(x) {
     return `${x}%`;
 }
 
-function setStandardsMapping(data, type) {
+function setStandardsMapping(data, key, type) {
     const mapping = {};
-    data.results.forEach((result) => {
+    data[key].results.forEach((result) => {
         const standardId = result.aggregationKeys[0].id;
+        if (!data.complianceStandards.some(({ id }) => id === standardId)) {
+            return; // because it implies standardId has hideScanResults: true
+        }
```

Comment:
I'm fine with the programming idiom, but the antecedent to "it" in the comment is a little opaque.

I think you mean, "If there's not at least one result for the standard in the result list, the standard must be hidden." ?